docs/wiki/EL/el-specs.md

```diff
@@ -0,0 +1,1319 @@
+# Execution Layer Specifications
+
+> **Where is it specified?**
+>
+> - [Yellow Paper paris version 705168a – 2024-03-04](https://ethereum.github.io/yellowpaper/paper.pdf) (note: This is outdated does not take into account post merge updates)
+> - [Python Execution Layer specification](https://ethereum.github.io/execution-specs/)
+> - EIPs [Look at Readme of the repo](https://github.com/ethereum/execution-specs)
+
+
+The core of the Ethereum Execution Layer is tasked with executing three types of transactions: legacy, Type 1, and Type 2. These transactions are processed by the quasi-Turing complete Ethereum Virtual Machine (EVM), which allows for virtually any computation, bounded only by gas constraints. This capability positions the EVM as a decentralized world computer, enabling decentralized applications (DApps) to run and transactions to be securely recorded on an immutable ledger. Beyond transaction processing, the EL is instrumental in storing data structures within the State DB. This not only facilitates the observation of current and historical states but also supports the Consensus Layer in block creation and validation. In this analogy, the Execution Layer acts as the CPU, with the Consensus Layer serving as the hard drive. Additionally, the EL defines the parameters of Ethereum's economic model, laying the foundation for blockchain operations.
+
+
```

Comment:
One quick comment - please, standardize spaces between paragraphs. Sometimes you leave one, two or three new lines between headers and new paragraphs. Let's keep the same in whole doc with 1 dividing new line between paragraphs or headers